app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -187,6 +191,8 @@ class HomeFragment : Fragment() {
         val activity = activity as HomeActivity
         val components = requireComponents
 
+        menuButton = view.findViewById<MenuButton>(R.id.menuButton)
```

Comment:
nit: I don't think you need to use findViewById. Synthetics should let you find the view in your `onConfigurationChanged` call itself.

Reply:
Agreed.  That is much cleaner.  Thanks I'll update.